tidb-cloud/limited-sql-features.md

```diff
@@ -60,7 +60,6 @@ TiDB Cloud works with almost all workloads that TiDB supports, but there are som
 | `SHOW PLUGINS` | Supported | Not supported [^8] |
 | `SHOW PUMP STATUS` | Not supported [^7] | Not supported [^7] |
 | `SHUTDOWN` | Not supported [^4] | Not supported [^4] |
-| `CREATE TABLE ... AUTO_ID_CACHE` | Supported | Not supported [^12] |
```

Comment:
@baiyuqing, Does this change mean `AUTO_ID_CACHE` is now supported on TiDB Serverless? If yes, please also remove the following from https://docs.pingcap.com/tidb/dev/sql-statement-create-table in this PR.

![image](https://github.com/pingcap/docs/assets/79440533/094cbe9e-a213-4ce3-bdc5-c3397737560e)

Reply:
Thank you for the reminder. Serverless supports `AUTO_ID_CACHE` now. I will edit the file you mentioned.